external-crates/move/crates/move-compiler/src/expansion/path_expander.rs

```diff
@@ -882,6 +936,7 @@ impl PathExpander for LegacyPathExpander {
                 PV::ModuleAccess(sp!(ident_loc, single_entry!(name, tyargs, is_macro)))
                     if self.aliases.module_alias_get(&name).is_some() =>
                 {
+                    self.ide_autocomplete_suggestion(context, LegacyPositionKind::Module, loc);
```

Comment:
I'm a bit confused about where these hooks are added. 
I would have figured it would be more natural to just track/clone out this information when entering a new scope? Not when building the paths up

Reply:
For clarity, I would have expected all of these hooks to be in `push_alias_scope`

Reply:
As discussed, this is the sort of "best" solution we have at the moment.